public/Census_Distribution_across_Buildings/utils.py

```diff
@@ -0,0 +1,188 @@
+    df = df.rename(columns={df.columns[1]: 'cnt'})  # Rename population column to 'cnt'
+    table_path = 's3://fused-asset/infra/census_bg_us'
+    print(df['GEOID'] ,  "there are the geoIDS")
+    if suffix:
+        table_path += f'_{suffix}'
+    print("meow", df.columns)
+    gdf = table_to_tile(bbox, table_path, use_columns=['GEOID', 'geometry'], min_zoom=12)
+    gdf['h3_index'] = gdf['geometry'].apply(lambda x: h3.latlng_to_cell(x.centroid.y, x.centroid.x, 11))
+    print(gdf)
+
+
+    if len(gdf)>0:
+        gdf2 = gdf.merge(df)
+        return gdf2
+    else:
+        print('No geometry is intersecting with the given bbox.')
+        return gpd.GeoDataFrame({})
+    
+@fused.cache
+def acs_5yr_meta(short=True): 
+    import pandas as pd
+    #Filter only records with cencus block groups data
+    tmp = pd.read_excel('https://www2.census.gov/programs-surveys/acs/summary_file/2021/sequence-based-SF/documentation/tech_docs/ACS_2021_SF_5YR_Appendices.xlsx')
+    table_ids_cbgs = tmp[tmp['Geography Restrictions'].isna()]['Table Number']
+    #Get the list of tables and filter by only totals (the first row of each table)
+    df_tables = pd.read_csv('https://www2.census.gov/programs-surveys/acs/summary_file/2022/table-based-SF/documentation/ACS20225YR_Table_Shells.txt', delimiter='|')
+    if short:
+        df_tables2 = df_tables.drop_duplicates('Table ID')
+    else:
+        df_tables2 = df_tables
+        df_tables2['Full Title']=df_tables2['Label']+' | '+df_tables2['Title']+' | '+df_tables2['Unique ID']
+    df_tables2 = df_tables2[df_tables2['Table ID'].isin(table_ids_cbgs)]
+    print(df_tables2, df_tables)
+    return df_tables2 
+
+
+@fused.cache
+def acs_5yr_table(tid, year=2022):
+    import pandas as pd
+    url=f'https://www2.census.gov/programs-surveys/acs/summary_file/{year}/table-based-SF/data/5YRData/acsdt5y{year}-{tid.lower()}.dat'
+    return pd.read_csv(url, delimiter='|')
+
+def search_title(title):
+    df_meta=acs_5yr_meta() 
+    #search for title in the list of tables 
+    search_column = 'Title' #'Title' #'Topics'
+    meta_dict = df_meta[['Table ID', search_column]].set_index(search_column).to_dict()['Table ID']
+    List = [[meta_dict[i], i] for i in meta_dict.keys() if title.lower() in i.lower()]
+    print(f'Chosen: {List[0]}\nfrom: {List[:20]}')
+    return List[0][0]
+
+
+import geopandas as gpd
+
+@fused.cache
+def get_census(bbox, census_variable='Total Pop', scale_factor=200, is_density=True, year=2022):
+    from utils import acs_5yr_bbox
+    import h3
+    import shapely
+    from shapely import Point, Polygon
+    #different geometry details per zoom level
+    if bbox.z[0]>12:
+        suffix=None
+    elif bbox.z[0]>10:
+        suffix='simplify_0001'
+    elif bbox.z[0]>8:
+        suffix='simplify_001'
+    elif bbox.z[0]>5:
+        suffix='simplify_01'
+    else:
+        suffix='centroid'
+    print(suffix)
+
+    #read the variables
```

Comment:
```suggestion
    # Read the variables
```
Please standardize all comments to use sentence case and have a space between the `#` and first letter, and add this to the `contributing` md.